project.clj

```diff
@@ -7,7 +7,7 @@
                  [clj-http              "2.0.0" :exclusions [org.clojure/clojure]]
                  [clojurewerkz/support  "1.1.0" :exclusions [com.google.guava/guava]]
                  ;; used by the native client
-                 [org.elasticsearch/elasticsearch "1.7.3"]]
+                 [org.elasticsearch/elasticsearch "2.0.0"]]
```

Comment:
There are additional breaking changes in 2.2. This commit only gets 2.0 support working.